model/src/main/kotlin/config/PackageConfiguration.kt

```diff
@@ -25,6 +25,7 @@ import org.ossreviewtoolkit.model.Identifier
 import org.ossreviewtoolkit.model.Provenance
 import org.ossreviewtoolkit.model.VcsInfo
 import org.ossreviewtoolkit.model.VcsType
+import org.ossreviewtoolkit.utils.stripCredentialsFromUrl
```

Comment:
Please add a reason to the commit message.

Reply:
Done.